insights/parsers/corosync_cmapctl.py

```diff
@@ -0,0 +1,64 @@
+"""
+CorosyncCmapctl - Command ``corosync-cmapctl [params]``
+=======================================================
+
+This module parses the output of the ``corosync-cmapctl [params]`` command.
+"""
+
+from insights import parser, CommandParser
+from insights.parsers import SkipException, ParseException
+from insights.specs import Specs
+
+
+@parser(Specs.corosync_cmapctl)
+class CorosyncCmapctl(CommandParser, dict):
+    """
+    Class for parsing the `/usr/sbin/corosync-cmapctl [params]` command.
+    All lines are stored in the dictionary with the left part of the equal
+    sign witout parenthese info as the key and the right part of equal sign
+    as the value.
+
+    Typical output of the command is::
+
+        config.totemconfig_reload_in_progress (u8) = 0
+        internal_configuration.service.0.name (str) = corosync_cmap
+        internal_configuration.service.0.ver (u32) = 0
+        internal_configuration.service.1.name (str) = corosync_cfg
+        internal_configuration.service.1.ver (u32) = 0
+        internal_configuration.service.2.name (str) = corosync_cpg
+        internal_configuration.service.2.ver (u32) = 0
```

Comment:
Btw, don't you think the unit (e.g. u32) is necessary?

Reply:
> Btw, don't you think the unit (e.g. u32) is necessary?

Yes, it is mentioned in the Jira ticket too. The info is just the type of value. so no need to store it.